pythonzm_Ops_task_views_41.py

```
@@ -0,0 +1,412 @@
+import datetime
+import os
+import json
+import zipfile
+import shutil
+from django.shortcuts import render
+from task.utils.gen_resource import GenResource
+from django.http import JsonResponse, HttpResponse
+from task.models import *
```

Ruff (0.13.1)

`from task.models import *` used; unable to detect undefined names

(F403)